nextjs-app/app/sitemap.ts

```diff
@@ -0,0 +1,24 @@
+import { MetadataRoute } from "next";
+
+export default async function sitemap(): Promise<MetadataRoute.Sitemap> {
+  const baseUrl =
+    process.env.NEXT_PUBLIC_BASE_URL ?? "https://https://thementorship.tw/";
```

Comment:
Seems that the default base URL repeats `https://` twice; it should be `https://thementorship.tw/`.

Reply:
PM will stick to use original one